mssqlcli/mssqlqueries.py

```diff
@@ -80,72 +86,77 @@ def get_view_columns():
             )   AS ist
             ON ist.table_name = isc.table_name AND ist.table_schema = isc.table_schema
             ORDER BY 1, 2'''
+    return normalize(sql)
 
 
 def get_views():
     """
     Query string to retrieve all views.
     :return: string
     """
-    return '''
+    sql = '''
```

Comment:
We can't add a ` u ` string prefix and assume unicode?

Reply:
I can put `u` in front of `'''` for making unicode. But we still need `normalize()` method for removing unintended spaces.